scripts/aggregates.py

```diff
@@ -4,52 +4,49 @@
 import numpy as np
 
 
-def get_L(n, omega_SS):
-    L = (omega_SS * n).sum()
+def get_L(n_s, omega_SS):
+    '''
+    Returns labor (a scalar)
+    '''
+    L = (omega_SS[1:] * n_s[1:]).sum() # indexing from 1 since the summation starts at E + 1, not E
 
     return L
 
+def get_K(b_s, omega_SS, imm_rates_SS):
+    '''
+    b = vector of savings of length S - 1
 
-def get_C(c, omega_SS):
-    C = (omega_SS * c).sum()
-
-    return C
-
-
-def get_Y(K, L, params):
-    alpha, A = params
-    Y = A * (K ** alpha) * (L ** (1 - alpha))
+    Returns capital (a scalar)
+    '''
+    K = (omega_SS[:-1] * b_s + imm_rates_SS[1:] * omega_SS[1:] * b_s).sum()
 
-    return Y
+    return K
 
 
-def get_I(K, Kp1, b, params):
-    (delta, g_n_SS, omega_SS, imm_rates_SS, g_y) = params
-    capital_flow = ((1 + g_n_SS) * np.exp(g_y) * (imm_rates_SS[1:] *
-                    omega_SS[1:] * b).sum())
+def get_C(c, omega_SS):
+    C = (omega_SS[1:] * c[1:]).sum() # indexing from 1 since the summation starts at E + 1, not E
 
-    I = ((1 + g_n_SS) * np.exp(g_y) * Kp1 - (1.0 - delta) * K -
-         capital_flow)
+    return C
 
+def get_I(K, K_sp1, delta):
+    # Need to think about how we're getting Kp1 - something analogous to what we did for b_sp_1 in class I presume
+    I = K_sp1 - (1 - delta) * K
     return I
 
 
-def get_K(b, params):
+def get_Y(b_s, C, I, omega_SS, imm_rates_SS):
     '''
-    b = (S-1,) vector of savings
+    Returns total goods consumed (a scalar)
     '''
-    (g_n_SS, omega_SS, imm_rates_SS) = params
-    K = ((1 / (1 + g_n_SS)) * (omega_SS[:-1] * b + imm_rates_SS[1:] *
-                               omega_SS[1:] * b).sum())
 
-    return K
+    Y = C + I - (imm_rates_SS[1:] * omega_SS[1:] * b_s).sum()
```

Comment:
You have `imm_rates_SS[1:]` here, should this just be `imm_rates_SS` if it is a scalar?

Reply:
Based on the above comment, disregard this.